languages/en-US/economy/slots.json

```
@@ -1,8 +1,8 @@
 {
-    "DESCRIPTION": "The Casino of Atlanta",
+    "DESCRIPTION": "The Atlanta casino",
     "USAGE": "{{prefix}}slots [amount]",
     "EXAMPLES": "{{prefix}}slots\n{{prefix}}slots 10",
     "DEFEAT": "**{{username}}** used {{money}} credit(s) and lost everything.",
     "VICTORY": "**{{username}}** used {{money}} credit(s) and won {{won}} credit(s)!",
-    "NOT_ENOUGH": "You do not have {{money}} credit(s)."
+    "NOT_ENOUGH": "You need at least `{{money}}` credit(s)."
```

Ce n'est pas plus beau en bold?
```suggestion
    "NOT_ENOUGH": "You need at least **{{money}}** credit(s)."
```